packages/logger/tests/writers.test.ts

```diff
@@ -0,0 +1,74 @@
+import { describe, expect, it } from 'vitest';
+import { Logger } from '../src/logger';
+import { ConsoleLogWriter, ConsoleLogWriterOptions } from '../src/writers';
+import { stableError } from './utils';
+
+describe.skipIf(
+  // bun is serialising the snapshots differently. the object keys are out of order...
```

Comment:
`fast-json-stringify` doesn't work with this?

Reply:
It does, it's just that bun serialises the snapshots differently putting the object keys in different order than node.

Reply:
I actually meant can we use that library to serialize the objects then take the snapshot to make it consistent between test runners?

Reply:
https://github.com/graphql-hive/gateway/pull/1030/files/a39189a656671ed9ab571bd39edba5c94524410d#r2103070734